corehq/apps/domain/decorators.py

```diff
@@ -319,7 +331,8 @@ def _inner(request, domain, *args, **kwargs):
             domain_name, domain = load_domain(request, domain)
             if not domain:
                 raise Http404()
-            if not request.couch_user.is_domain_admin(domain_name):
+            if not _page_is_whitelist(request.path, domain_name) and \
```

Comment:
I think this clause also needs a `user.is_superuser` clause otherwise whitelisted pages will be accessible for any user.